app.js

```diff
@@ -105,19 +105,72 @@ function renderProductsToPage() {
 };
 
 //renders results to page at conclusion of survey
+//NOW UTILIZES Charts.js LIBRARY!!!
 function renderResultsToPage() {
-  var unorderedEl = document.createElement('ul');
+  var ctx = document.getElementById('chart').getContext('2d');
+  var pieCtx = document.getElementById('pie-chart').getContext('2d');
+
+  var barData = [];
+  var pieData = [];
+
+  var labelColors = ['red', 'orange', 'yellow', 'green', 'blue', 'indigo', 'violet', 'red', 'orange', 'yellow', 'green', 'blue', 'indigo', 'violet', 'red', 'orange', 'yellow', 'green', 'blue', 'indigo'];
+
+  var labels = [];
 
   for (var i = 0; i < products.length; i++) {
-    var resultsMessage = products[i].numberOfClicks + ' votes for ' + products[i].productName;
+    products[i].percentage = products[i].getPercentage();
+    pieData.push(products[i].percentage);
 
-    var listItemEl = document.createElement('li');
-    listItemEl.textContent = resultsMessage;
+    barData.push(products[i].numberOfClicks);
 
-    unorderedEl.appendChild(listItemEl);
+    labels.push(products[i].productName);
   }
 
-  chartEl.appendChild(unorderedEl);
+  var barChartData = {
+    type: 'bar',
+    data: {
+      labels: labels,
+      datasets: [{
+        label: labels,
```

Comment:
you are sending your full array names as the label here, which is why when you hover over the bar it displays all of those names.  Try giving setting this label to something like "number of votes" and see if you like it.